.builds/freebsd.yml

```
@@ -0,0 +1,23 @@
+image: freebsd
```

This is not a valid image?
Try one of these:
```suggestion
image: freebsd/latest
```
```suggestion
image: freebsd/13.x
```
```suggestion
image: freebsd/12.x
```
```suggestion
image: freebsd/11.x
```

---

Yep, I realized, going with 12.x